### PR TITLE
Modify doc

### PR DESCRIPTION
**Pull request by Linexl**
#7762

### CODE REVIEW

File: documentation20/en/12.taos-sql/docs.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # TAOS SQL
 
-TDengine provides a SQL-style language, TAOS SQL, to insert or query data. To read through this document, you should have some basic understanding about SQL.
+TDengine provides a SQL-style language, TAOS SQL, to insert or query data, and the document supports other common tips. To finish this document, readers should have basic understanding about SQL.
```
Comment (glzhao89) [inline documentation20/en/12.taos-sql/docs.md]: "...query data. This document introduces TAOS SQL and supports ...."

Comment (glzhao89) [reply documentation20/en/12.taos-sql/docs.md]: I think we should keep the "read through". it has clearer meaning than "finish".

File: documentation20/en/12.taos-sql/docs.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-TAOS SQL is the main way for users to write and query data to TDengine. TAOS SQL is similar to standard SQL to facilitate users to get started quickly. Strictly speaking, TAOS SQL is not and does not attempt to provide SQL standard syntax. In addition, since TDengine does not provide deletion function for time-series data, the relevant function of data deletion is non-existent in TAO SQL.
+TAOS SQL is the main tool for users to write and query data into TDengine. TAOS SQL provides a syntax style similar to standard SQL to facilitate users to get started quickly. Strictly speaking, TAOS SQL is not and does not attempt to provide SQL standard syntax. In addition, since TDengine does not provide deletion functionality for temporal structured data, the data relevant functionalities of data deletion is non-existent in TAO SQL.
```
Comment (glzhao89) [inline documentation20/en/12.taos-sql/docs.md]: we should better use "into/from" here, meaning "write data into TDengine" and "query data from TDengine". The description has different directions.

Comment (glzhao89) [reply documentation20/en/12.taos-sql/docs.md]: we should change "the data relevant functionality" to "the relevant functions". "functions" does not have the same meaning as "functionalities" here, as this indicates sql functions can do mathematic calculations like avg(), sum() etc..

**Review by glzhao89 — CHANGES_REQUESTED** [review]

**Review by glzhao89 — APPROVED** [review]

**Comment by Eliacy** [comment]
LGTM

**Pull request merged**